backend/src/API/mailAPI.ts

```diff
@@ -63,6 +84,14 @@ const emailAdmins = async (req: Request, subject: string, text: string) => {
   });
 };
 
+/**
+ * Send an email to a member
+ * @param req - The request made when sending the email
+ * @param member - The member being sent the email
+ * @param subject - The subject of the email
+ * @param text - The body of the email
+ * @returns - The information of the member being sent the email
```

Comment:
How about for `@returns`, we write it as

"The response body containing information of the...". Same thing for the two docstrings down below.